content/docs/payment-providers/quickpay/changelog.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Changelog
+description: Changelog for the QuickPay Payment Provider for Vendr, the eCommerce solution for Umbraco v8+
+---
+
+## v1.0.0 (Unreleased) 
+**Date:** TBC
+**Description:** Initial release of the QuickPay Payment Provider.
+--- 
+
+<changelog>
+<changelog-group category="Added">  
+
+    
+* Added QuickPay V10 Payment Provider
+
+
+</changelog-group>
+</changelog>
```

Comment:
These aren't needed for the initial commit

Reply:
Yeah, it was just added in DIBS so I just included it as well, but I guess you want to remove it from DIBS, since it instead has the description.